packages/manager/modules/manager-pci-common/src/api/data/image.ts

```diff
@@ -0,0 +1,23 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+
+export type TImage = {
+  creationDate: string;
+  flavorType?: string;
+  id: string;
+  minDisk: number;
+  minRam: number;
+  name: string;
+  planCode: string;
+  region: string;
+  size: number;
+  status: string;
```

Comment:
Could we put a list of possible values instead of string ?

Reply:
the status is a string in the api